src/visual/aria.ts

```diff
@@ -154,6 +154,7 @@ export default function ariaVisual(ecModel: GlobalModel, api: ExtensionAPI) {
         }
 
```

Comment:
Can we move the code here before checking `description`, so that it doesn't need to be set in two different situations?